src/Symfony/Component/DependencyInjection/Compiler/ReplaceAliasByActualDefinitionPass.php

```diff
@@ -32,6 +32,7 @@ class ReplaceAliasByActualDefinitionPass extends AbstractRecursivePass
      */
     public function process(ContainerBuilder $container)
     {
+        $this->enableExpressionProcessing();
```

Comment:
Are you sure about that ? Replacing a reference inside the expression won't change the expression anyway. It will keep using the alias.